src/main/java/jenkins/metrics/impl/JenkinsMetricProviderImpl.java

```diff
@@ -342,46 +358,47 @@ protected Integer transform(NodeStats value) {
+                        new DerivativeGauge<JobStats, Double>(jobStats) {
                             @Override
                             protected Double transform(JobStats value) {
                                 return value.getDepthAverage();
                             }
                         }),
                 metric(name("jenkins", "project", "count"),
-                        new AutoSamplingHistogram(new DerivativeGauge<JobStats,Integer>(jobStats) {
+                        new AutoSamplingHistogram(new DerivativeGauge<JobStats, Integer>(jobStats) {
                             @Override
                             protected Integer transform(JobStats value) {
                                 return value.getProjectCount();
                             }
                         }).toMetricSet()),
-                metric(name("jenkins", "project", "enabled","count"),
-                        new AutoSamplingHistogram(new DerivativeGauge<JobStats,Integer>(jobStats) {
+                metric(name("jenkins", "project", "enabled", "count"),
+                        new AutoSamplingHistogram(new DerivativeGauge<JobStats, Integer>(jobStats) {
                             @Override
                             protected Integer transform(JobStats value) {
                                 return value.getEnabledProjectCount();
                             }
                         }).toMetricSet()),
-                metric(name("jenkins", "project", "disabled","count"),
-                        new AutoSamplingHistogram(new DerivativeGauge<JobStats,Integer>(jobStats) {
+                metric(name("jenkins", "project", "disabled", "count"),
+                        new AutoSamplingHistogram(new DerivativeGauge<JobStats, Integer>(jobStats) {
                             @Override
                             protected Integer transform(JobStats value) {
                                 return value.getDisabledProjectCount();
                             }
                         }).toMetricSet()),
-                metric(name("jenkins", "job", "queuing", "duration"), (jenkinsJobQueueTime = new Timer())),
-                metric(name("jenkins", "job", "waiting", "duration"), (jenkinsJobWaitingTime = new Timer())),
-                metric(name("jenkins", "job", "blocked", "duration"), (jenkinsJobBlockedTime = new Timer())),
-                metric(name("jenkins", "job", "buildable", "duration"), (jenkinsJobBuildableTime = new Timer())),
-                metric(name("jenkins", "job", "building", "duration"), (jenkinsJobBuildingTime = new Timer())),
-                metric(name("jenkins", "job", "total", "duration"), (jenkinsJobTotalTime = new Timer())),
+                metric(name("jenkins", "job", "queuing", "duration"), (jenkinsJobQueueDuration = new Timer())),
+                metric(name("jenkins", "task", "queuing", "duration"), (jenkinsTaskQueueDuration = new Timer())),
```

Comment:
this is a new one that holds the old value... because the old value was incorrect, being the time tasks spent queuing and the number of tasks queuing... now `jenkins.job.queuing.duration` is just jobs